### PR TITLE
Fix a few tests

### PR DESCRIPTION
**Pull request by cammarosano**
Some test cases got a few details lost in translation... (blame and see!)

### CODE REVIEW

File: addons/html_editor/static/tests/editor/unbreakable/delete.test.js [hunk 1]
```diff
@@ -468,72 +467,66 @@ describe("forward", () => {
...
-                //     stepFunction: deleteBackward,
-                //     contentAfter:
-                //         '<div>abc</div><ol><li>def</li><li><div><div>[]ghi</div></div></li><li>jkl</li></ol>',
-                // });
-            }
-        );
+        test("should not outdent while nested within a list item if the list is unbreakable", async () => {
+            // Only one LI.
+            await testEditor({
+                contentBefore: '<p>abc</p><ol t="1"><li>[]def</li></ol>',
```
Comment (FrancoisGe) [inline addons/html_editor/static/tests/editor/unbreakable/delete.test.js:485]: what is t=1 ?

File: addons/html_editor/static/tests/editor/unbreakable/delete.test.js [hunk 2]
```diff
@@ -468,72 +467,66 @@ describe("forward", () => {
...
+        });
     });
     describe("selection not collapsed", () => {
+        // @phoenix @todo: this spec is weird. I would argue in favor of
+        // <p class="oe_unbreakable">a[]</p><ol><li>ef</li><li>ghi</li></ol>
         test.todo("shoud not merge list item in the previous unbreakable sibling (1)", async () => {
             await testEditor({
                 contentBefore: unformat(`
-                        <p>a[bc</p>
+                        <p class="oe_unbreakable">a[bc</p>
```
Comment (FrancoisGe) [inline addons/html_editor/static/tests/editor/unbreakable/delete.test.js:529]: Why we don t need the class before ?

File: addons/html_editor/static/tests/editor/unbreakable/delete.test.js (hunk 1 above)
Comment (cammarosano) [reply addons/html_editor/static/tests/editor/unbreakable/delete.test.js:485]: No idea... I guess the idea was to trigger this condition in `isUnbreakable`: `node.hasAttribute("t") `
Anyway, the original test had this, and somehow it disappeared when Nicolas moved stuff. See https://github.com/odoo-dev/odoo/commit/cbfa43786ec8669ef93e98e4e2178e43beb09833#r140126285

File: addons/html_editor/static/tests/editor/unbreakable/delete.test.js (hunk 2 above)
Comment (cammarosano) [reply addons/html_editor/static/tests/editor/unbreakable/delete.test.js:529]: The original test had the class. When Nicolas reorganized the tests, for some reason the class disappeared. See https://github.com/odoo-dev/odoo/commit/cbfa43786ec8669ef93e98e4e2178e43beb09833#r140126370

Comment (cammarosano) [reply addons/html_editor/static/tests/editor/unbreakable/delete.test.js:529]: (without the class, the tests does not match its description...)

File: addons/html_editor/static/tests/editor/unbreakable/delete.test.js (hunk 1 above)
Comment (FrancoisGe) [reply addons/html_editor/static/tests/editor/unbreakable/delete.test.js:485]: ok

File: addons/html_editor/static/tests/editor/unbreakable/delete.test.js (hunk 2 above)
Comment (FrancoisGe) [reply addons/html_editor/static/tests/editor/unbreakable/delete.test.js:529]: ok

**Pull request merged**